pkg/event/event_ce_test.go

```diff
@@ -36,7 +37,7 @@ var (
 	uriLocation = "http://localhost:9090/event/subscription/1234"
 	endPointURI = "http://localhost:8080/event/ack/event"
 	resource    = "/cluster/node/ptp"
-	_type       = "ptp_status_type"
+	_type       = ptp.STATUS
```

Comment:
remove this default ptp.STATUS 
it should be valid status from the API 
example  `ptp.EventPtpStateChange `

Reply:
done